### ISSUE TITLE
Account recovery request

### ISSUE THREAD

**Issue by RogerMarsh** [pypi/support#4406]
### PyPI Username

`rogermarsh`

### Reason for request

I seem to have logged in successfully but get the message:

"Your primary email address (<redacted>) is unverified. (Verify your email or add a new address).

On clicking 'Verify ... address' I see 'Email sent' in the Security History but so far no email in my inbox: nor do I see a way to add a new address.

My previous login, before today, was back in December 2023 (maybe November), so I was expecting maybe a 2FA rejection.  Back then I did verify I could set up and use 2FA on TestPyPI.  About to check TestPyPi is still ok so I know what to expect on activating 2FA for PyPI to do uploads.

Just did final check of spam at my ISP before clicking 'Submit new issue'.

### Recovery Codes

- [X] I never generated or have lost access to the recovery codes for my account

### Code of Conduct

- [X] I agree to follow the PSF Code of Conduct

### Acknowledgement

- [X] I understand that it may take a significant amount of time to process my account recovery request.

**Comment by RogerMarsh** [comment]
Upload to testpypi succeeded.  But no prompts for username or password as advertised at https://packaging.python.org/en/latest/tutorials/packaging-projects/# so I tried upload to pypi which failed due to unverified primary email address.

Adding -vvv option to twine command revealed username and password are taken from configuration file.  On inspection .pypyrc has entries for both pypi and testpypi.

So it seems I did set up 2FA for pypi, otherwise why is there a pypi entry.

**Comment by RogerMarsh** [comment]
I have two accounts on testpypi.  One since about the time my pypi account was created and has not been set to 2FA.  the other setup within the last year to try out 2FA.

Next time I looked at email inbox after posting previous comment I saw an email from TestPyPI <noreply@test.pypi.org> saying the older (non-2FA) account needed conversion to 2FA.

This email is addressed to the unverified primary email address.

Nice there is a reminder about the unconverted account; but I guess there is no point expecting the promised email for the pypi account to ever arrive.
